q2_demux/plugin_setup.py

```diff
@@ -125,7 +125,10 @@
     function=q2_demux.summarize,
     inputs={'data':
             SampleData[SequencesWithQuality | PairedEndSequencesWithQuality]},
-    parameters={'n': qiime2.plugin.Int},
+    parameters={
+      'n': qiime2.plugin.Int,
+      'delta': qiime2.plugin.Int
```

Comment:
Or just leave the summarize declaration thing in the style of `n`:

````PYTHON
def summarize(output_dir: str, data: _PlotQualView,
              n: int=10000, delta: int=1) -> None:
````

And then do the magic in `plugin_setup`:

````PYTHON
parameters={
      'n': qiime2.plugin.Int % Range(0, None),
      'delta': qiime2.plugin.Int % Range(0, None)
    },
````